include/windows/osd.h

```diff
@@ -694,6 +694,12 @@ static inline SOCKET ofi_socket(int domain, int type, int protocol)
 	return socket(domain, type, protocol);
 }
 
+static inline int ofi_setsockopt(SOCKET fd, int level, int optname,
```

Comment:
@shefty It was introduced to avoid conversation from any types to (const char *) optval on Windows.